python-api/tests/unittests/test_processmonitor_unit.py

```diff
@@ -0,0 +1,223 @@
+"""Unittests for ProcessMonitor class in server.py"""
```

Comment:
Awesome, thanks for writing tests for the testing infrastructure! It seems a bit silly but it really helps make sure we have confidence that the tests actually work.